tests/foreman/destructive/test_clone.py

```diff
@@ -105,6 +109,8 @@ def test_positive_clone_backup(target_sat, sat_ready_rhel, backup_type, skip_pul
         ).status
         == 0
     )
+    # Assert clone won't fail due to BZ
+    assert sat_ready_rhel.execute('satellite-clone --help').status == 0
```

Comment:
@Griffin-Sullivan @evgeni I have already covered this scenario in https://github.com/SatelliteQE/robottelo/pull/10853/files#diff-0136ad9f3a21adf5716d5013efd837789362d2624fb5c83f55a226650e0c0470R131 Even though it mentions [BZ#2170034](https://bugzilla.redhat.com/show_bug.cgi?id=2170034) but it's actually covering scenario for [BZ#2142514](https://bugzilla.redhat.com/show_bug.cgi?id=2142514). 
Also, `satellite-clone --help` command didn't reproduce the issue but `satellite-clone --list-tasks` did. 

@Griffin-Sullivan can you update `test_positive_list_tasks` instead and add the customer bz there?

Reply:
Huh, I would have expected (and pretty sure that's how I tested the change) that `--help` has the same issues as `--list-tasks` but down the road either one works as they are both a) quick and b) don't require any customer data to identify a problem.

Given the test is already present, I think we can close this one.

(as for BZ numbers: they are the same "issue", just on different pythons)